fierce/fierce.py

```diff
@@ -168,12 +168,12 @@ def default_expander(ip):
 
 
 def traverse_expander(ip, n=5):
-    class_c = get_class_c_network(ip)
-
-    result = [ipaddress.IPv4Address(ip + i) for i in range(-n, n + 1)]
-    result = [i for i in result if i in class_c]
+    ip = int(ip)
+    class_c_floor = ip - (ip % 256)
 
-    return result
+    ip_min = max(ip - n, class_c_floor)
+    ip_max = min(ip + n, class_c_floor + 255)
```

Comment:
Could we move `class_c_floor + 255` up into a variable `class_c_ceiling`? I'm a fan of documenting constants/magic numbers with variable names :)

Reply:
Makes sense, will do.